src/components/select/simple-select/simple-select.pw.tsx

```diff
@@ -996,6 +997,93 @@ test.describe("SimpleSelect component", () => {
     });
   });
 
+  ([
+    ["top", "100px 300px", 150, 2],
+    ["bottom", "0px 300px", 150, 68],
+    ["top-end", "100px 300px", 150, 2],
+    ["bottom-end", "0px 300px", 150, 68],
+    ["top-start", "100px 0px", 0, 2],
+    ["bottom-start", "100px 0px", 0, 168],
+  ] as [
+    NonNullable<SimpleSelectProps["listPlacement"]>,
+    string,
+    number,
+    number
+  ][]).forEach(([position, margin, left, top]) => {
+    test(`should render list wider than the input when listWidth is 350 and positioned based on listPlacement of ${position}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <ListWidth margin={margin} listPlacement={position} listWidth={350} />
+      );
+
+      const positionValue =
+        !position.includes("end") && !position.includes("start")
+          ? `${position}-end`
+          : position;
+      await selectText(page).click();
+      const listElement = selectListPosition(page);
+      const { inputWidth } = await selectText(page).evaluate((el) => ({
+        inputWidth: window
+          .getComputedStyle(el.parentElement as HTMLElement)
+          .getPropertyValue("width"),
+      }));
+      const { listLeftPosition, listTopPosition } = await listElement.evaluate(
+        (el) => ({
+          listLeftPosition: el.getBoundingClientRect().left,
+          listTopPosition: el.getBoundingClientRect().top,
+        })
+      );
+
+      await expect(listElement).toHaveAttribute(
+        "data-floating-placement",
+        positionValue
+      );
+      await expect(listElement).toBeVisible();
+      await expect(listElement).toHaveCSS("width", "350px");
+      expect(inputWidth).toBe("200px");
+
+      // sub-pixel rendering means this is not always exact
+      expect(listLeftPosition).toBeLessThanOrEqual(left);
+      expect(listTopPosition).toBeLessThanOrEqual(top);
+      expect(listLeftPosition).toBeGreaterThanOrEqual(left - 1);
+      expect(listTopPosition).toBeGreaterThanOrEqual(top - 1);
```

Comment:
**suggestion(non-blocking)**: another option we could consider here is using the `toBeCloseTo` assertion, which is designed for handling this kind of comparison. Not essential though - just something that might be helpful:

```suggestion
      expect(listLeftPosition).toBeCloseTo(left);
      expect(listTopPosition).toBeCloseTo(top);
      expect(listLeftPosition).toBeCloseTo(left);
      expect(listTopPosition).toBeCloseTo(top);
```

Reply:
Good point I'll update this before I merge